### DISCUSSION TITLE
Oracle support

### DISCUSSION THREAD

**Discussion by avico78** [collerek/ormar#185] [General]
As ormar build on top sqlalechamy and seems sqlalchemy does support Oracle DB :
https://www.sqlalchemy.org/features.html
is there a chance/plan to include Oracle support also for ormar?

**Comment by collerek** [comment]
The limitation comes from `databases` (that runs the quries) not from `sqlalchemy` (that build the queries syntax etc.).

But as I said in #142 the databases is like half-abandoned by their maintainers so in the long run I will probably switch as soon as the API stabilizes (it's quite a big refactor and `async` support is still in alpha, there was also some kind of ugly api for async but don't remember exactly as I checked it some time ago). 

So in general - definitely yes (preferably all sqlalchemy backends supported), but it can take some time.
